src/main/scala/net/cakesolutions/CakePublishMavenPlugin.scala

```diff
@@ -168,7 +168,7 @@ object CakePublishMavenPlugin extends AutoPlugin {
         require(
           gitStatus.isSuccess && gitStatus.getOrElse("").isEmpty,
           s"Aborting release process for ${name.value}. Project should be a " +
-            "clean git repository."
+            s"clean git repository. Error output:\n$gitStatus"
```

Comment:
I wonder if it renders the `git status ...` command output when it fails or a big `Failure({exception message})` thing? An example of the final message would be helpful.